region/tourian/main/Metroid Room 3.json

```diff
@@ -205,22 +224,25 @@
       "requires": [
         {"enemyKill": {
           "enemies": [["Metroid", "Metroid", "Metroid"]],
-          "excludedWeapons": ["Super", "Missile"]
+          "explicitWeapons": ["PowerBombPeriphery"]
```

Comment:
5PBs works really well from when I was testing.  The idea is closer to doing 3+3, but incidental damage nearly kills the distant metroids as they come close.  4 is often enough, but 5 isn't rare if one does get caught.

And group vs kill separately: the health cost gets high fast when grouping plus you basically need to know how to run into them to group them, which is a lot like knowing how to avoid them.